setup.py

```diff
@@ -44,7 +44,8 @@
     "url": "https://github.com/XanaduAI/StrawberryFields",
     "license": "Apache License 2.0",
     "packages": find_packages(where="."),
-    "package_data": {"strawberryfields": ["backends/data/*", "apps/data/*"]},
+    "package_data": {"strawberryfields": ["backends/data/*", "apps/data/feature_data/*",
+                                          "apps/data/sample_data"]},
```

Comment:
How come `apps/data/feature_data/*` with asterisk is listed, but `apps/data/sample_data` does not include an asterisk?

Reply:
```suggestion
                                          "apps/data/sample_data/*"]},
```
Good spot!